pages/home-page.ts

```diff
@@ -0,0 +1,16 @@
+import { Page } from '@playwright/test';
+import {boxedStep} from '../util/boxed-step';
+
+export class HomePage {
+    constructor(private readonly page: Page) {}
+
+    async goto() {
+        await this.page.goto('https://magento.softwaretestingboard.com/');
```

Comment:
Need to setup base URL from configs in case that on a project you can have multiple testing environments like QA, STAGE, PROD, etc.